src/controllers/chatController.js

```diff
@@ -0,0 +1,114 @@
+import { ApiError } from '../exceptions/api.error.js';
+import { chatService } from '../service/chat.service.js';
+import { EventEmitter } from 'events';
+import { wss } from '../index.js';
+
+const emitter = new EventEmitter();
+
+const createRoom = async (req, res) => {
+  const { name } = req.body;
+
+  if (!name) {
+    throw ApiError.badRequest('Name is empty');
+  }
+
+  await chatService.createRoom(name);
+
+  const roomId = await chatService.getId(name);
+
+  res.status(200).json({ roomId });
+};
+
+const messages = async (req, res) => {
+  const { text } = req.body;
+  const { roomId } = req.params;
+
+  const refreshToken = req.headers.cookie;
+
+  if (!refreshToken) {
+    throw ApiError.badRequest('Refresh token not found');
```

Comment:
Consider providing more informative error messages. For example, 'Refresh token not found' could specify where it was expected to be found.